pkg/apis/servicecatalog/types.go

```diff
@@ -127,10 +127,9 @@ type ServiceClass struct {
 	Plans         []ServicePlan
 	PlanUpdatable bool // Do we support this?
 
-	// OSB-specific
-	// OSBGUID is the identity of this object for use with the OSB API.
+	// ExternalID is the identity of this object for use with the OSB API.
 	// Immutable.
-	OSBGUID string
+	ExternalID string
```

Comment:
On today's call we agreed to set metadata.Name to the GUID, and ExternalID to the GUID as well. This would allow us to twiddle the metadata.Name field to something human readable later if we could ever figure out how w/o losing the ExternalID/GUID value.  We should add a DisplayName to hold the human readable name though.